src/Network/Receive/ServerType0.pm

```diff
@@ -2020,6 +2020,7 @@ sub item_skill {
 sub map_changed {
 	my ($self, $args) = @_;
 	$net->setState(4);
+	%pet = ();
```

Comment:
hey, i might saying nonsense...

but why this needs to stay here? i mean, is already declared at Globals.pm isn't?

Reply:
oh now i understand, is to clear old values right? sorry for misunderstanding

Reply:
Also this should be in functions.pl, not here

Reply:
@lututui , the pet ID changed every map server , iRO

my current bot date years ago, work with addition only on functions.pl
but not with our latest bot, so i need to put it on map_changed,looks like something changed

Reply:
But when map changes the method on functions.pl is always called... Or am I wrong?